### DISCUSSION TITLE
how to target bootstrap in php files?

### DISCUSSION THREAD

**Discussion by ghost** [twbs/bootstrap#38064] [Q&A]
I'm having an issue regarding importing bootstrap libraries in my PHP FILE.

**Comment by julien-deramond** [comment]
Hey @Sehrish24 
Please don't upload zip file for security reason. Could you provide a [CodePen](https://codepen.io) (or something else) to share your code? Or directly a part of it in Markdown maybe?
_Edited your description to remove the zip file_

**Comment by denizgolbas** [comment]
```html
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
```

Put this in beetween head tags.

**Comment by yasser-acc** [comment]
# How to Target Bootstrap in PHP Files


1. Download the latest version of Bootstrap from the official website: https://getbootstrap.com/.
2. Unzip the downloaded file and copy the "css" and "js" folders to your project directory.
3. In your PHP file, add the following code to include the Bootstrap CSS file in the head section of your HTML:

   ```html
   <head>
       <link rel="stylesheet" href="css/bootstrap.min.css">
   </head>
4. Similarly, add the following code to include the Bootstrap JavaScript file at the end of your HTML body:
 ```
<body>
    
    <script src="js/bootstrap.min.js"></script>
</body> 
```
5. Once you've included the Bootstrap files in your PHP file, you can start using Bootstrap classes and components to style your HTML elements.
